tests/console/system_prepare.pm

```diff
@@ -38,6 +38,19 @@ sub run {
         }
     }
 
+    # Register the modules after media migration, only for sle<15
+    # poo#54131 [SLE][Migration][SLE12SP5]test fails in system_prepare -
+    # media upgrade need add modules after migration
+    if (get_var('SCC_ADDONS') && is_sle('<15') && get_var('MEDIA_UPGRADE')) {
+        assert_script_run 'SUSEConnect --url ' . get_required_var('SCC_URL') . ' -r ' . get_required_var('SCC_REGCODE');
+        my $myaddons = get_var('SCC_ADDONS');
+        # After media upgrade, system don't include ltss extension
+        $myaddons =~ s/ltss,?//g;
+        if ($myaddons ne '') {
+            register_addons_cmd($myaddons);
+        }
```

Comment:
Please use SUSEConnect --list-extensions to show all extensions and modules actived for later debug.